decks.js

```diff
@@ -825,6 +825,103 @@ echo $message;`,
                 seenHint: false
             }
         ]
+    },
+    {
+        name: "Sports",
+        instructions: "The emoji/s on each card are hints for the sport. You can flip (click) the card to view additional clues. Drag and drop the card onto the matching text clue on the right, which will alias the sport name in easy mode. You can make a maximum of 3 wrong matches.",
+        cards: [
+            {
+                name: "Archery",
+                content: {
+                    secondary: "R🍒"
+                },
+                hint: "More than the bow and arrows.",
+                matchValue: `bow and arrow`,
+                seenHint: false
+            },
+            {
+                name: "Football",
+                content: {
+                    secondary: "🦶⚽"
+                },
+                hint: "11 Players, 1 Heartbeat.",
+                matchValue: `goooaaalll!`,
+                seenHint: false
+            },
+            {
+                name: "Cricket",
+                content: {
+                    secondary: "😭😺"
+                },
+                hint: "bat and ball game played",
+                matchValue: `Batting Gloves and batting helmet`,
+                seenHint: false
+            },
+            {
+                name: "Golf",
+                content: {
+                    secondary: "⭕F"
```

Comment:
Could you think of a different emoji sequence here? This is since "gol" would only be understood by Hindi speakers, and I would like most categories to be accessible to all :)